app/models/service_template_terraform_template.rb

```diff
@@ -17,11 +17,28 @@ def self.create_catalog_item(options, _auth_user)
 
     transaction do
       create_from_options(options).tap do |service_template|
+        dialog_ids = service_template.send(:create_dialogs, config_info)
```

Comment:
Why use `send` here?  This method doesn't appear to be private and the method name isn't dynamic so I think you can just
```suggestion
        dialog_ids = service_template.create_dialogs(config_info)
```

Reply:
This came from ansible  implementation,  myself  was puzzled  why  'send' ?   
simply did not fix what was not broken

Reply:
:+1: That's because their create_dialogs instance method is private, that isn't a good reason to use send but that is why it is needed there and not here.